tracking/visualize.py

```diff
@@ -0,0 +1,94 @@
+import os
+import sys
+import argparse
+
+
+
+prj_path = os.path.join(os.path.dirname(__file__), '..')
+if prj_path not in sys.path:
+    sys.path.append(prj_path)
+
+from lib.test.evaluation.data import SequenceList, Sequence
+from lib.test.utils.load_text import load_text
+from lib.test.evaluation import get_dataset
+from lib.test.evaluation.running import run_dataset
+from lib.test.evaluation.tracker import Tracker
+import numpy as np
+import shutil
+
+class TrackDataset():
+
+    def __init__(self, dir_name):
+
+        self.sequence_list = os.listdir(dir_name)
+        self.base_path = dir_name
+
+    def get_sequence_list(self):
+        has_folders = 0
+        for f in self.sequence_list:
+            if os.path.isdir(os.path.join(self.base_path, f)):
+                has_folders = 1
+                break
+        if has_folders == 1:
+            return SequenceList([self._construct_sequence(s) for s in self.sequence_list])
+        return SequenceList([self._construct_sequence('')])
+
+    def _construct_sequence(self, sequence_name):
+        anno_path = '{}/{}/groundtruth.txt'.format(self.base_path, sequence_name)
+
+        ground_truth_rect = load_text(str(anno_path), delimiter=',', dtype=np.float64)
+
+        frames_path = '{}/{}'.format(self.base_path, sequence_name)
+        frame_list = [frame for frame in os.listdir(frames_path) if frame.endswith(".jpg")]
+        frame_list.sort(key=lambda f: int(f[:-4]))
+        frames_list = [os.path.join(frames_path, frame) for frame in frame_list]
+
+        return Sequence(sequence_name if sequence_name != '' else self.base_path.split('\\')[-1], frames_list, 'got10k', ground_truth_rect.reshape(-1, 4))
```

Comment:
I suggest using `os.sep` and os.path.join()` or something instead of hardcoded slashes